cli/bundle-install.go

```diff
@@ -79,13 +79,25 @@ When using this command, you must provide the path to the file, as follows:
   gort bundle install [flags] config_path
 
 Flags:
-  -h, --help   Show this message and exit
+  -h, --help			Show this message and exit
+  -e, --enable			Automatically enable a bundle after installing?
+						[default: False]
+  -f, --force			Install even if a bundle with the same version is
+						already installed. Applies only to bundles
+						installed from a file, and not from the Warehouse
+						bundle registry. Use this to shorten iteration
+						cycles in bundle development.  [default: False]
 
 Global Flags:
   -P, --profile string   The Gort profile within the config file to use
 `
 )
 
+var (
+	flagEnable bool
```

Comment:
Can you namespace these vars a little better? Compare with `user-update.go`:

```go
var (
	flagUserUpdateEmail    string
	flagUserUpdateName     string
	flagUserUpdatePassword string
)
```

Reply:
Absolutely, done.